pizza_cutter/coadd_sim_slicer/slicer.py

```diff
@@ -489,40 +504,112 @@ def _build_object_data(
     return output_info
 
 
-def _read_data(
+def _load_psf(*, fname):
+    """
+    load the psf object. Currently only supports PSFEx
+
+    Parameters
+    ----------
+    fname: string
+        Path name for the psf file
+
+    Returns
+    --------
+    The psf object, currently this will be a PSFEx object
+    """
+    return psfex.PSFEx(fname)
+
+
+def _load_fits_objects(
         *,
         image_path, image_ext,
         bkg_path, bkg_ext,
         weight_path, weight_ext,
         seg_path, seg_ext,
-        bmask_path, bmask_ext,
-        psf):
-    """Read the data.
+        bmask_path, bmask_ext):
+    """
+    Load fits objects for the input paths
+
+    If paths are the same for some objects, the same underlying fits object is
+    used
 
     Returns
     -------
-    img, bkg, seg, wgt, bmask, pex, wcs
+    FITS HDU objects for each type, or None as appropriate
```

Comment:
We should keep the ordering in the docs here so that ppl know what is coming out.

Reply:
"People" being me and/or you in like 6 months...

Reply:
I pushed a fix